### ISSUE TITLE
Compilation warnings

### ISSUE THREAD

**Issue by rmoorman** [slime-lang/slime#105]
Adding slime to a project and compiling it using elixir 1.3 (1.3.2 to be precise) gives a lot of warnings like

```
warning: the variable "tag" is unsafe as it has been set inside a case/cond/receive/if/&&/||. Please explicitly return the variable value instead. For example:

    case int do
      1 -> atom = :one
      2 -> atom = :two
    end

should be written as

    atom =
      case int do
        1 -> :one
        2 -> :two
      end

Unsafe variable found at:
  lib/slime/parser.ex:72
```

[This gist](https://gist.github.com/rmoorman/693e14f783fe7914c89baa88818258fb) includes the full output of the compilation using a dummy project.

**Comment by doomspork** [comment]
Thanks for the report @rmoorman!  We'll need to update Slime and supporting libraries for 1.3, want to take a shot at it? 😀

**Comment by rmoorman** [comment]
@doomspork, I just tried to use the github master branch directly instead of the version published on hex and the warnings are gone :-). So the next hex release will do away with the warnings presumably. Sorry for the noise and keep up the good work.

**Comment by rmoorman** [comment]
was solved by #102

**Comment by doomspork** [comment]
Thank you @rmoorman, we'll cut a new version 👍
